app/jobs/scheduled/turn_off_do_not_disturb.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Jobs
+  class TurnOffDoNotDisturb < ::Jobs::Scheduled
```

Comment:
Would it be more efficient to schedule a job each time a user sets DND vs. running a check every minute?